src/popup/pages/MultisigProposalDetails.vue

```diff
@@ -439,7 +443,11 @@ export default defineComponent({
 
         proposalCompleted.value = true;
       } catch (error) {
-        handleUnknownError(error);
+        if (isInsufficientBalanceError(error)) {
+          openModal({ name: MODAL_DEFAULT, icon: 'alert', msg: root.$t('modals.vaultLowBalance.msg') });
+        } else {
+          handleUnknownError(error);
```

Comment:
We can notify the user if there is any other error by using a modal.
I am okay with the current approach too. Up to you @pstachel

Reply:
You're right, I'll adjust the code